cmd/account/mgmt/account-unassign.go

```diff
@@ -0,0 +1,370 @@
+	if err != nil {
+		return err
+	}
+
+	for _, m := range accessKeys.AccessKeyMetadata {
+
+		inputDelKey := &iam.DeleteAccessKeyInput{
+			AccessKeyId: m.AccessKeyId,
+			UserName:    &user,
+		}
+		_, err = o.awsClient.DeleteAccessKey(inputDelKey)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (o *accountUnassignOptions) deleteSigningCert(user string) error {
+
+	inputListCert := &iam.ListSigningCertificatesInput{
+		UserName: &user,
+	}
+	cert, err := o.awsClient.ListSigningCertificates(inputListCert)
+	if err != nil {
+		return err
+	}
+
+	for _, c := range cert.Certificates {
+		inputDelCert := &iam.DeleteSigningCertificateInput{
+			CertificateId: c.CertificateId,
+			UserName:      &user,
+		}
+		_, err = o.awsClient.DeleteSigningCertificate(inputDelCert)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (o *accountUnassignOptions) deletePolicies(user string) error {
+	inputListPolicies := &iam.ListUserPoliciesInput{
+		UserName: &user,
+	}
+	policies, err := o.awsClient.ListUserPolicies(inputListPolicies)
+	if err != nil {
+		return err
+	}
+
+	for _, p := range policies.PolicyNames {
+		inputDelPolicies := &iam.DeleteUserPolicyInput{
+			PolicyName: p,
+			UserName:   &user,
+		}
+		_, err = o.awsClient.DeleteUserPolicy(inputDelPolicies)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (o *accountUnassignOptions) deleteAttachedPolicies(user string) error {
+	inputListAttachedPol := &iam.ListAttachedUserPoliciesInput{
+		UserName: &user,
+	}
+	attachedPol, err := o.awsClient.ListAttachedUserPolicies(inputListAttachedPol)
+	if err != nil {
+		return err
+	}
+
+	for _, ap := range attachedPol.AttachedPolicies {
+		inputDetachPol := &iam.DetachUserPolicyInput{
+			PolicyArn: ap.PolicyArn,
+			UserName:  &user,
+		}
+		_, err = o.awsClient.DetachUserPolicy(inputDetachPol)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (o *accountUnassignOptions) deleteGroups(user string) error {
+
+	inputListGroups := &iam.ListGroupsForUserInput{
+		UserName: &user,
+	}
+	groups, err := o.awsClient.ListGroupsForUser(inputListGroups)
+	if err != nil {
+		return err
+	}
+
+	for _, g := range groups.Groups {
+		inputRemoveFromGroup := &iam.RemoveUserFromGroupInput{
+			GroupName: g.GroupName,
+			UserName:  &user,
+		}
+		_, err = o.awsClient.RemoveUserFromGroup(inputRemoveFromGroup)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (o *accountUnassignOptions) deleteUser(user string) error {
+	inputDelUser := &iam.DeleteUserInput{
+		UserName: &user,
+	}
+	_, err := o.awsClient.DeleteUser(inputDelUser)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (o *accountUnassignOptions) untagAccount(id string) error {
+	inputUntag := &organizations.UntagResourceInput{
+		ResourceId: &id,
+		TagKeys: []*string{
+			aws.String("owner"),
+			aws.String("claimed"),
+		},
+	}
+	_, err := o.awsClient.UntagResource(inputUntag)
+	if err != nil {
```

Comment:
You forgot to test this situation. Please add this as part of your unit-tests ;)